code/style.css

```diff
@@ -1,58 +1,140 @@
+* { 
+  box-sizing: border-box;
+ }
+ *:before,
+ *:after {
+  box-sizing: border-box;
+ }
+
+html,
+body {
+ height: 100%;
+ position: relative;
+}
+
 body {
   margin: 0;
-  font-family: "Courier New", Courier, monospace;
+  font-family: GabrielWeissFriendsFont;
+  background-image: url(./assets/background.jpg);
+  background-size: cover;
 }
 
-p {
-  margin: 1px;
-  font-weight: 300;
-  background-color: #fff;
+.main-container {
+  min-height: 100vh; 
+  overflow: hidden;
+  display: block;
+  position: relative;
+  padding-bottom: 50px; 
+}
+
+
+h1 {
+  font-size: 80px;
+  margin-top: 10px;
+  margin-bottom: 0;
+  font-family: GabrielWeissFriendsFont;
+  color: #000000;
+  text-align: center;
+}
+
+h2 {
+  margin-top: 0;
+  font-family: GabrielWeissFriendsFont;
+  color: #000000;
```

Comment:
We can write it shorter: `#000`;

Reply:
Yess, I know, and I am going to fix that, but I love memorizing colors this way.